examples/SDM_example.py

```diff
@@ -0,0 +1,204 @@
+
+
+from __future__ import absolute_import, print_function
+
+import array
+import threading
+import logging
+import time
+
+from ant.easy.node import Node
+from ant.easy.channel import Channel
+from ant.base.message import Message
+from ant.base.commons import format_list
+
+
+import struct
+import sys
+
+# Definition of Variables
+NETWORK_KEY= [0xb9, 0xa5, 0x21, 0xfb, 0xbd, 0x72, 0xc3, 0x45]
+Device_Type = 124  # 124 = Stride & Distance Sensor
+Device_Number = 12345  # Change if you need.	
+Channel_Period = 8134
+Channel_Frequency = 57	
+
+# Fictive Config of Treammill 
+TreadmillSpeed = 2.777  # m/s => 10km/h
+TreadmillCadence = 160
+		
+##########################################################################
+
+class AntSendDemo:
+
+    def __init__(self):
+
+        self.ANTMessageCount = 0
+        self.ANTMessagePayload = array.array('B', [0, 0, 0, 0, 0, 0, 0, 0])
+
+		# Init Variables, needed
+        self.LastStrideTime = 0
+        self.StridesDone = 0
+        self.DistanceAccu = 0
+        self.Speed_Last = 0
+        self.TimeRollover = 0
+		
+        self.TimeProgramStart = time.time()
+        self.LastTimeEvent = time.time()
+
+	# Building up the Datapages
+		# This is just for demo purpose and can/will look diverent for every implementation
+    def Create_Next_DataPage(self):
+        # Define Variables
+        UpdateLatency_7 = 0
+		
+        self.ANTMessageCount += 1
+		
+        # Time Calculations
+        self.ElapsedSseconds = time.time() - self.LastTimeEvent
+        self.LastTimeEvent = time.time()	
+        UpdateLatency_7 += self.ElapsedSseconds # 1Second / 32 = 0,03125
+        UL_7 = int(UpdateLatency_7 / 0.03125)
+
+        # Stride Count, Accumulated strides.
+        # This value is incremented once for every two footfalls. 
+        StrideCountUpValue = 60.0 / (TreadmillCadence / 2.0) # In our Example 0,75
+		#print("LastStrideTime: ", self.LastStrideTime)	# jst for DEBUGGING
+        while (self.LastStrideTime > StrideCountUpValue):
+            self.StridesDone += 1
+            self.LastStrideTime-=StrideCountUpValue
+        self.LastStrideTime += self.ElapsedSseconds
+        if (self.StridesDone > 255):
+            self.StridesDone -= 255
+
+        # DISTANCE
+		# Accumulated distance, in m-Meters, Rollover = 256
+        self.DistanceBetween =  self.ElapsedSseconds * TreadmillSpeed 
+        self.DistanceAccu+=self.DistanceBetween # Add Distance beetween 2 ANT+ Ticks to Accumulated Distance
+        if (self.DistanceAccu > 255):
+            self.DistanceAccu -= 255
+        
+        self.distance_H = int(self.DistanceAccu) # just round it to INT
+        self.DistanceLow_HEX = int((self.DistanceAccu-self.distance_H)*16)
+        # erstmal zum Checken
+		#if(DistanceLow_HEX > 15) { printf("ERROR: Distance LOW HEX ueber 15: %d \n", DistanceLow_HEX); }
+        #printf("=> Distanz m: %d,%d \n", distance_H, DistanceLow_HEX)

```

Comment:
There are a few blocks of code commented out here, either remove or use them to do something.

Reply:
yeahh... Sorry, just forgot to cleanup my debugging.

done now